Modulefile

```diff
@@ -1,5 +1,5 @@
 name    'puppet-scout'
-version '2.0.4'
+version '2.0.7'
```

Comment:
Thanks. Sorry I missed that before. It's sad that we have to specify this in two places.